awot/io/read_t28.py

```diff
@@ -0,0 +1,158 @@
+from __future__ import absolute_import, print_function
+import numpy as np
+from netCDF4 import num2date
+from .flight import _winduv
+
+
+def read_t28_netcdf(ncFile):
+    """
+    T-28 has enough differences from other aircraft netCDF files to
+    warrant its own read script.
+
```

Comment:
Looking at your code below, it appears to do the same stuff (mostly) as the updated netcdf reader. If we were able to generalize a couple of parts of the netcdf reader it *should* work.

I  ordered and tried some data from the JPOLE experiment. The encouraging thing is that it appears that they use the same global attribute I mention above. But:
The reader bombs in these two spots.
1) I currently use the filename to pull out the dimension name and data acquisition rate. I'm not sure of a robust way to deal with this. We process the UWKA data anywhere between 1 - 50 Hz with many dimension included depending on if we downsample data.

2) Different ```time``` variables names. I guess we add a time variable flag that specifies which variable to use, otherwise attempt some common variables. The issue arises because of legacy code keeping things like ```time``` ```Time``` ```HHMMSS``` alive so that **old** code didn't need to be rewritten. We have some code here that has dust blown off it every few years and compiled and used.